RELEASE_NOTES.md

```diff
@@ -172,14 +172,18 @@ Then run the below command, and replace all the configuration with the one that
 bundle exec rake decidim:webpacker:install
 ```
 
-This will make the necessary changes in the `config/webpacker.yml`, but also in the `config/webpack/` folder.
+This will make the necessary changes in the `config/shakapacker.yml`, but also in the `config/webpack/` folder.
+
+As of [\#11728](https://github.com/decidim/decidim/pull/11728), Shakapacker has been upgraded to version 7.1.0, which will stop using `config/webpacker/test.js`, `config/webpacker/development.js`, `config/webpacker/production.js`.
+
+The project no longer needs to use `config/webpacker.yml`, in favor of `config/shakapacker.yml`. It is recommended to remove the `config/webpacker.yml` to avoid further confusion.
 
```

Comment:
Same here:

```console
mv config/webpacker.yml config/shakapacker.yml
```

Reply:
Is not very clear for me what is expected from me.

Reply:
To add this command as an instruction in the RELEASES_NOTES. For me its much more clear to implementers that they need to actually run this command instead of having to parse and understand instructions (specially when we have lots of instructions already!)

Reply:
Now i understand what are you referring to. 
I have simplified the release notes, and they are like this: 

![image](https://github.com/decidim/decidim/assets/105683/7bfd468f-95e2-4341-9bba-80ef43c453c9)

Reply:
OK, I'll do a full review of the Releases notes in another PR to make it consistent and easy to follow. In my experience when we add that many instructions without easy to copy and paste comments, this generates situations where implementers don't actually know what to do/don't do those actions.
In this case, they at least would see warnings, but that's when people start to ignore too many warnings and so on.